parm/stage/master_gfs.yaml.j2

```diff
@@ -108,7 +108,7 @@
     {% set COMOUT_OCEAN_RESTART_PREV_MEM = COM_OCEAN_RESTART_TMPL | replace_tmpl(previous_cycle_dict) %}
     {% set COMOUT_OCEAN_ANALYSIS_MEM = COM_OCEAN_ANALYSIS_TMPL | replace_tmpl(current_cycle_dict) %}
     {% set COMOUT_MED_RESTART_PREV_MEM = COM_MED_RESTART_TMPL | replace_tmpl(previous_cycle_dict) %}
-    {% set COMOUT_WAVE_RESTART_PREV_MEM = COM_WAVE_RESTART_TMPL | replace_tmpl(previous_cycle_and_run_dict) %}
```

Comment:
If `previous_cycle_and_run_dict` is not going to be used here then lines 97-101 should also be removed. Only the wave IC staging was using it based on the pre-refactored staging job behavior and the ICs on disk.

This seems to be another cold-start quirk, which wasn't captured in the CI tests at the time of refactoring. Thanks for resolving.

Reply:
Removed